packages/widget/src/utils/explorerLink.ts

```diff
@@ -0,0 +1,26 @@
+import { config } from "@/constants/wagmi";
+import { chains } from "@/state/chains";
+
+export const createExplorerLink = ({ chainID, txHash, chainType }: { chainID: string, chainType?: string, txHash: string }) => {
+  if (!chainType) return txHash
+  switch (chainType) {
+    case "cosmos":
+      {
+        const chain = chains.find((chain) => chain.chain_id === chainID);
+        const mintscan = chain?.explorers?.find((explorer) => explorer.kind === "mintscan");
+        const explorer = mintscan || chain?.explorers?.[0];
+        const url = explorer?.tx_page?.replace("${txHash}", txHash);
+        return url || txHash
+      }
+    case "evm": {
+      const chain = config.chains.find((chain) => chain.id === Number(chainID));
+      const url = chain?.blockExplorers?.default.url
+      return url ? `${url}/tx/${txHash}` : txHash
+    }
+    case "svm": {
+      return `https://solscan.io/tx/${txHash}`
+    }
+    default:
+      return txHash
+  }
+}
```

Comment:
@codingki it doesn't seem obvious to me that `txHash` can be returned from this function,
I think we should either remove `txHash` from here and pass it separately to `ErrorPageTimeout`, or rename this function to make it clear that the `txHash` is potentially returned, something like "getExplorerLinkOrTxHash"